project/build.scala

```diff
@@ -1,113 +0,0 @@
-import sbt._
-
-object Builds extends sbt.Build {
-  import Keys._
-  import ls.Plugin.{lsSettings,LsKeys}
-  import sbtbuildinfo.Plugin._
-  import conscript.Harness.conscriptSettings
-
-  val g8version = "0.6.9-SNAPSHOT"
-
-  lazy val buildSettings = lsSettings ++ Seq(
```

Comment:
It appears that the `lsSettings` weren't carried over—if discontinuing publishing metadata to ls was a conscious decision because ls might be dead (#179), I think we could additionally remove some cruft:

- The `ls-sbt` plugin that's being loaded.
- JSON metadata files in `app/src/main/ls`, `library/src/main/ls`, etc.
- Concatenate the files in `notes` into one `CHANGELOG.md`?

This is a separate matter from removing the library support for ls version lookups, which is the subject of #179.